app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,9 @@
+class MovieActorsController < ApplicationController
```

Comment:
Awesome controller organization! Using the create action in this joins controller is a very RESTful architecture